CanonicalLtd/snappy-docs
# Make a self-hosting Jekyll project

Because of the transition to Kubernetes, this repository now needs to become a self-hosting web project, like all our others. This is outlined in a bit more depth in https://github.com/canonical-websites/docs.ubuntu.com/issues/88.

This ports over the Jekyll templates, sass and configuration infrasture and the ./run script from [snapcraft.io-static-pages](https://github.com/canonical-websites/snapcraft.io-static-pages/), with minor modifications.

Resolves #110

QA
--

``` bash
./run
```

Browse to <http://127.0.0.1:8004> and check it looks the same as <https://snapcraft.io/docs>, and that internal links still works.
